modules/katamari/src/test/ts/atomic/api/fun/FunTest.ts

```diff
@@ -133,3 +134,42 @@ UnitTest.test('Check call :: apply(constant(a)) === undefined', () => {
     Assert.eq('eq', x, hack);
   }));
 });
+
+describe('katamari.Fun.pipe', () => {
```

Comment:
This should be a describe with `katamari.Fun` as the root and then have an inner `context` for `pipe`.

- [ ] Break this up into nested suites so it'll convert better

Reply:
Done